docs/dev/setup.rst

```diff
@@ -39,6 +39,13 @@ Quickstart
 
 2. From the root of the repository, run::
 
+     $ npm install
```

Comment:
Can we run this as part of `make build`?

Please also remove the warning below about localhost:8000

Reply:
Yes, at the cost of adding a couple of seconds to every `make build` call as `npm` will need to check that everything that should be installed is installed.

Going to presume that you meant the note about `localhost:3000`, which I'll remove.

Reply:
I think a couple of seconds is fine for a command that's not used very often.

3000, yeah.

Reply:
D'oh, I had already set it up to run as a part of `make build`. So I just removed this useless addition to the docs.